src/components/Guidelines/GuidelinesContent.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent } from "react";
+import ReactMarkdown from "react-markdown";
+import { AccordionItem } from "../../components/UI/Accordion/AccordionItem";
+import { importAll } from "../../common/utils/importAll";
+import { getSortedByDateAsc } from "../../utils";
+import remarkGfm from "remark-gfm";
```

Comment:
what do we need from remarkGFM that is not supported from ReactMarkdown?

Reply:
my bad, should not have this package inside, will double check again